app/assets/javascripts/contacts.js

```diff
@@ -43,27 +45,38 @@ var showSuccess = function(contact, method) {
   );
 };
 
-var erredContacts = function(_xhr, status, error) {
+var recordError = function(xhr, status, error, message) {
```

Comment:
Minor: `recordError` seems very "passive" to me. If I saw this function name I would assume it was just tracking it and not that anything was presented to the user.